frontend/smart-kickers-game/src/components/Game/GameStatistics/GameStatistics.js

```diff
@@ -22,6 +35,9 @@ function GameStatistics({ finalScores, onNewGameRequested }) {
         <div className="table-item">{finalScores.blue}</div>
         <div className="table-item">score</div>
         <div className="table-item">{finalScores.white}</div>
+        {data?.FastestShot && <div className="table-item">{returnFastestShot(TeamID.Team_blue)}</div>}
+        <div className="table-item">fastest shot of the game</div>
+        {data?.FastestShot && <div className="table-item">{returnFastestShot(TeamID.Team_white)}</div>}
```

Comment:
I run the app without the backend and went to the `Game statistics` section. This is what's displayed. I don't think it should look like that.
![image](https://user-images.githubusercontent.com/56163767/187888805-cc59a1ac-f5fb-443d-8bf8-d6e55e16698f.png)